[READY] add total hours to report

mischaboldy opened the pull request.

jurre commented on app/controllers/reports_controller.rb:
```diff
@@ -5,6 +5,10 @@ def index
+    @total_hours = @entries.sum(:hours)
+    @total_billed= @entries.where(billed: true).sum :hours
+    @total_unbilled = @entries.where(billed: false).sum :hours
+
```
So we're passing a lot of different variables to our view here, it's usually a sign that we need an object to hold them.

All of these have to do with the entries, so maybe we could have an `EntriesCollection` object or something? It would look something like this:

```ruby
class EntriesCollection
  attr_reader :entries
  def initialize(entries)
    @entries = entries
  end

  def total_hours
    entries.sum(:hours)
  end

  def total_biller
    entries.where(billed: true).sum :hours
  end

  def total_unbilled
    entries.where(billed: false).sum :hours
  end
end
```

Then in our controller:

```
def index
  # ...
  @entries_collection = EntriesCollection.new(entries)
end
```

And in our view:

```haml
%td= @entries_collection.total_hours
```

mischaboldy commented:
reworked form mileages and hours, also added collection class @jurre (thanks for the suggestion). I ran into a problem with the mileages/hours and didn't know which option was better. Since there are 2 totals now i had a few options:

- two of each method, one for hours one for mileages in the collection class (chose this)
- two seperate classes, 1 for hours 1 for mileages
- 3 methods with a string parameter to determine total hours or mileages

Maybe there is another way i don't know of yet.

jurre commented:
We've decided to freeze feature development as none of the maintainers currently have time to properly steward contributions from the community.
